index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!--poppins font link-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,600;0,800;1,200;1,600&display=swap"
+      rel="stylesheet">
+    <!--fontawsome link-->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <!--styles.css link-->
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+  </head>
+
+  <body>
+    <nav>
+      <div class="logo-container">
+        <div class="logo">
+          <a href="index.html">My Logo</a>
+        </div>
+        <div class="bars">
+          <i class="fa fa-bars"></i>
+        </div>
+      </div>
+    </nav>
+
+    <header>
+      <section class="header">
+        <h1>I'm Anold. <br>Glad to see <br> you!</h1>
+        <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work
+          and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <article class="social">
+          <h3>LET'S CONNECT</h3>
+          <div class="social_icons">
+            <a href=""><img src="images/twitter.svg" alt="twitter icon"></a>
+            <a href=""><img src="images/Linkedin icon-1.svg" alt="Linkedin icon"></a>
+            <a href=""><img src="images/medium.svg" alt="medium icon"></a>
+            <a href=""><img src="images/github.svg" alt="github icon"></a>
+            <a href=""><img src="images/angellist.svg" alt="angellist icon"></a>
+          </div>
```

Comment:
- [ ] It is required  for the list of icons to use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.